src/modules/AddOnProduct.rb

```diff
@@ -2209,6 +2215,35 @@ def add_rename(old_name, new_name)
 
   private
 
+    # Returns a map containing renamed products
+    #
+    # The content of this map comes from:
+    #
+    # * libzypp (through #ResolvableProperties method)
+    # * PRODUCT_RENAMES_FALLBACK_MAP constant
+    #
+    # The information contained here can be extended (for example, with information
+    # from SCC) using the #add_rename method.
+    #
+    # @return [Hash] Renames following the format <old_name> => [ <new_name> ]
+    #
+    # @see #add_rename
+    # @see #renamed?
+    def product_renames
+      @product_renames = PRODUCT_RENAMES_FALLBACK_MAP.clone
+
+      products = Pkg.ResolvableDependencies("", :product, "")
+      products.each do |product|
+        next if product["replaces"].nil?
+        product["replaces"].each do |replaced|
```

Comment:
I think you should check the [obsolete](https://github.com/yast/yast-pkg-bindings/blob/3508f517bb93974c1484da2b4f0c124368935f60/src/Resolvable_Properties.cc#L141) key and look for the product names.

E.g. `sle-ha-geo-release` contains `Obsoletes:       product:sle-hae-geo`, see [line 51 here](https://build.suse.de/package/view_file/SUSE:SLE-12-SP2:GA/_product:sle-ha-geo-release/sle-ha-geo-release.spec?expand=1).